src/index.css

```diff
@@ -1,13 +1,149 @@
 body {
```

Comment:
don't use index.css file. inside your components create separate file and put the css code